start_pokealarm.py

```diff
@@ -140,9 +141,26 @@ def parse_settings(root_path):
                         help='Maximum number of attempts an alarm makes to send a notification.')
     parser.add_argument('-tz', '--timezone', type=str, action='append', default=[None],
                         help='Timezone used for notifications.  Ex: "America/Los_Angeles"')
+    parser.add_argument('--log_to_file',
+                        help=('Enable logging to file.'),
+                        action='store_true', default=False)
+    parser.add_argument('--log_path',
+                        help=('Defines directory to save log files to.'),
+                        default='logs/')
 
     args = parser.parse_args()
 
+    if args.log_to_file:
+        # Create directory for log files.
+        if not os.path.exists(args.log_path):
+            os.mkdir(args.log_path)
+        date = strftime('%Y%m%d_%H%M')
+        filename = os.path.join(args.log_path, '{}.log'.format(date))
+        fh = logging.FileHandler(filename)
+        fh.setFormatter(
+            logging.Formatter(fmt='%(asctime)s [%(processName)15.15s][%(name)10.10s][%(levelname)8.8s] %(message)s'))
+        logging.root.addHandler(fh)
+
```

Comment:
Have you tested potentially adding the ability to control the verbosity of resulting logs, such as giving the option to only log ERRORS, etc?

Reply:
Probably beyond the scope of this for now.